apps/dev/.env.local.example

```diff
@@ -0,0 +1,2 @@
+VERCEL_TEAM_ID=
```

Comment:
Is this file needed?

Reply:
It isn't needed, but it is a common pattern to add these example .env files to let the user know what variables are needed to run the project locally. This is because the `.env.local` file is not added to the source code.

Reply:
Fair enough!